alembic/env.py

```diff
@@ -0,0 +1,80 @@
+from logging.config import fileConfig
+
+from sqlalchemy import engine_from_config
+from sqlalchemy import pool
+
+from alembic import context
+
+from database import Base
+
+# this is the Alembic Config object, which provides
+# access to the values within the .ini file in use.
+config = context.config
+
+# Interpret the config file for Python logging.
+# This line sets up loggers basically.
+if config.config_file_name is not None:
+    fileConfig(config.config_file_name)
+
+# add your model"s MetaData object here
+# for "autogenerate" support
+# from myapp import mymodel
+# target_metadata = mymodel.Base.metadata
+target_metadata = [Base.metadata]
```

Comment:
The `target_metadata` is set to a list containing `Base.metadata`. If you have multiple metadata objects, this is correct. However, if you only have one, you can directly assign `Base.metadata` without wrapping it in a list.